Nim/Nim/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+
+class NimGame
+{
+    static void Main(string[] args)
+    {
```

Comment:
You don't need `class` and `static void Main` in any .NET project since .NET 5. Just remove the top 6 lines of code and the last two and it should still work as before :)